### DISCUSSION TITLE
Home Assistant OS struttura cartelle

### DISCUSSION THREAD

**Discussion by sky7176** [indomus/forum#1530] [HUB: Home Assistant]
Buona sera,
c'è una guida che spieghi come sono organizzate le cartelle di una installazione Home Assistant OS?
Mi spiego, se da terminale vado /config/custom_component, al suo interno trovo la cartella di nodered, hacs,argon40 ecc. ecc.
ma i componenti tipo google drive backup dove stanno?

Altra domanda ma HASSOS "gira" su docker?

Scusate la mia ignoranza, ma vorrei capire cosa c'è sotto il cofano di HA.

Giuseppe

**Comment by snakuzzo** [comment]
Ciao...perdonami ma quello che chiedi non è che si racconta in mezzo messaggio su un forum.
Se ti interessa, studia l'architettura [QUI](https://developers.home-assistant.io/docs/architecture_index/)

**Reply by sky7176** [reply]
grazie
